nessie/golden/lakectl_help.golden

```diff
@@ -29,13 +29,13 @@ Available Commands:
   tag             Create and manage tags within a repository
 
 Flags:
-      --base-uri string     base URI used for lakeFS address parse
-  -c, --config string       config file (default is $HOME/.lakectl.yaml)
-  -h, --help                help for lakectl
-      --log-format string   set logging output format
-      --log-level string    set logging level (default "none")
-      --log-output string   set logging output file
-      --no-color            don't use fancy output colors (default when not attached to an interactive terminal)
-  -v, --version             version for lakectl
+      --base-uri string      base URI used for lakeFS address parse
+  -c, --config string        config file (default is /home/barak/.lakectl.yaml)
```

Comment:
I bet this is by mistake :)

Reply:
```suggestion
  -c, --config string        config file (default is $HOME/.lakectl.yaml)
```

Reply:
> I bet this is by mistake :)

yes, the difference between `lakectl` vs `lakectl help`